tools/attrd_updater.c

```diff
@@ -205,6 +214,11 @@ static GOptionEntry addl_entries[] = {
       INDENT "value, or the value set by a later request).  Default is 'no'.",
       "UNTIL" },
 
+    { "utilization", 'z', G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, attr_set_type_cb,
+      "Use resource utilization attribute instead of instance attribute\n"
```

Comment:
The utilization terminology is a little confusing. This could be considered a resource utilization attribute, but we refer to it elsewhere as a node utilization attribute, so we should stick with that. It's worth mentioning that this will only be used if the attribute needs to be created, otherwise its existing type will continue to be used regardless. Also, bump the feature set.

FYI, I eventually want to use "capacity" for nodes and keep "utilization" for resources, but that's a bigger project for the future.

Reply:
Fixed, though I'm not especially happy with the new description.